pages/common/htpasswd.md

```diff
@@ -0,0 +1,23 @@
+# htpasswd
+
+> Create and manage htpasswd files to protect web server directories using basic authentication.
+
+- Create/overwrite htpasswd file:
+
+`htpasswd -c {{path/to/file}} {{user_name}}`
+
+- Add user to htpasswd file or update existing user:
+
+`htpasswd {{path/to/file}} {{user_name}}`
+
+- Add user to htpasswd file in batch mode without password verification (for script usage):
+
+`htpasswd -b {{path/to/file}} {{user_name}} {{password}}`
```

Comment:
"without password verification" may seem to contradict with the `{{password}}` token. Would "without an interactive password prompt" work better?